docroot/modules/humsci/hs_blocks/src/Plugin/Block/SocialMediaBlock.php

```diff
@@ -0,0 +1,305 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\hs_blocks\Plugin\Block;
+
+use Drupal\Core\Block\BlockBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+use Drupal\Core\Session\AccountProxyInterface;
+use Drupal\multivalue_form_element\Element\MultiValue;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Provides a social media block.
+ *
+ * @Block(
+ *   id = "hs_blocks_social_media_block",
+ *   admin_label = @Translation("Social Media Block"),
+ *   category = @Translation("H&S Blocks"),
+ * )
+ */
+final class SocialMediaBlock extends BlockBase implements ContainerFactoryPluginInterface {
+
+  /**
+   * The current user.
+   *
+   * @var \Drupal\Core\Session\AccountProxyInterface
+   */
+  protected $currentUser;
+
+  /**
+   * {@inheritdoc}
+   */
+  public function __construct(
+    array $configuration,
+    $plugin_id,
+    $plugin_definition,
+    AccountProxyInterface $current_user,
+    ) {
```

Comment:
Constructor parameters are typically left to one line despite the length. Also remove the comma after the last parameter.
```suggestion
  public function __construct(array $configuration, $plugin_id, $plugin_definition, AccountProxyInterface $current_user) {
```

Otherwise, change indentation:
```suggestion
  public function __construct(
    array $configuration,
    $plugin_id,
    $plugin_definition,
    AccountProxyInterface $current_user,
  ) {
```

Reply:
Fixed.